index.js

```diff
@@ -2,7 +2,10 @@
 var sliceAnsi = require('slice-ansi');
 var stringWidth = require('string-width');
 
-module.exports = function (input, columns) {
+module.exports = function (input, columns, options) {
+	options = options || {};
+	options.position = options.position || 'end';
```

Comment:
```js
var position = options.position || 'end';
```

Right now you're overwriting the user supplied object. And since objects are reference-based in JS, if the user calls `cliTruncate` with the same object multiple times the result will be different.